pkg/controller/nstemplateset/nstemplateset_controller_test.go

```diff
@@ -1222,4 +1422,42 @@ var (
 	username test.TemplateParam = `
 - name: USERNAME
   value: johnsmith`
+
+	advancedCrq test.TemplateObject = `
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    labels:
+      toolchain.dev.openshift.com/provider: codeready-toolchain
+      toolchain.dev.openshift.com/tier: advanced
+      toolchain.dev.openshift.com/owner: ${USERNAME}
+    name: for-${USERNAME}
+  spec:
+    quota:
+      hard:
+        limits.cpu: 1750m
+        limits.memory: 7Gi
```

Comment:
I guess that there is no difference between the `basicCrq` and `advancedCrq`. How about making the limits bigger for the `advancedCrq`

```suggestion
        limits.cpu: 2000m
        limits.memory: 10Gi
```